gcsfs/tests/test_inventory_report.py

```diff
@@ -184,6 +186,61 @@ def test_parse_raw_inventory_report_config_returns_correct_config(self):
         except Exception as e:
             pytest.fail(f"Unexpected exception: {e}.")
 
+    @pytest.mark.asyncio
+    async def test_fetch_inventory_report_metadata_no_reports(self, mocker):
+        
+        # Create a mock for GCSFileSystem.
+        gcs_file_system = mocker.MagicMock(spec=GCSFileSystem)
+
+        # Mock the _call method to return a page with two items
+        # and then a page with one item and without next page token.
+        gcs_file_system._call.side_effect = [{"items": [], "nextPageToken": None}]
+
+        # Create a mock for InventoryReportConfig.
+        inventory_report_config = mocker.MagicMock(spec=InventoryReportConfig)
+        inventory_report_config.bucket = "bucket_name"
+        inventory_report_config.destination_path = "destination_path"
+
+        # If no inventory report metadata is fetched, an exception should be raised.
+        with pytest.raises(ValueError) as e_info:
```

Comment:
1. Changed to using the `match` flag.
2. I guess we could do that, but since there is only 1 place so far doing this, setting a constant might stand out too much. I personally think it'd be better to conform with how the other tests are written ;)

Reply:
It's fine if we need to fix the test in the future, it forces us to consider why we are making that future change